vcbuild.bat

```diff
@@ -422,12 +432,56 @@ set USE_EMBEDDED_NODE_INSPECT=1
 goto node-tests
 
 :node-tests
-if "%test_args%"=="" goto cpplint
+if "%test_args%"=="" goto test-v8
 if "%config%"=="Debug" set test_args=--mode=debug %test_args%
 if "%config%"=="Release" set test_args=--mode=release %test_args%
 echo running 'cctest %cctest_args%'
 "%config%\cctest" %cctest_args%
 call :run-python tools\test.py %test_args%
+goto test-v8
+
+:test-v8
+if not defined custom_v8_test goto cpplint
+if not defined DEPOT_TOOLS_PATH goto depot-tools-not-found
+if "%config%"=="Debug" set test_args=%target_arch%.debug
+if "%config%"=="Release" set test_args=%target_arch%.release
+set savedpath=%path%
+set path=%DEPOT_TOOLS_PATH%;%path%
```

Comment:
@gibfahn  - I followed the syntax of environment variables defined outside the script to be all caps. e.g. `VS140COMNTOOLS`, `PROCESSOR_ARCHITEW6432`, etc. I see that some tooling related environment variables are also all caps, e.g. `GYP_MSVS_VERSION`. But again i am open to change if you strongly feel like that.